linux/nvdimm_cold_add_remove/nvdimm_cold_add_remove.yml

```diff
@@ -17,11 +17,13 @@
 
         - include_tasks: ../../common/skip_test_case.yml
           vars:
-            skip_msg: "{{ guest_os_ansible_distribution }} doesn't support NVDIMM"
+            skip_msg: "{{ vm_guest_os_distribution }} doesn't support NVDIMM"
             skip_reason: "Not Supported"
           when: >
             ('Flatcar' in guest_os_ansible_distribution or
-             guest_os_ansible_distribution == 'UnionTech')
+             guest_os_ansible_distribution in ['UnionTech', 'Uos', 'Amazon'] or
```

Comment:
Flatcar can be also put in the list.

Reply:
Flatcar's distribution might be changed from version to version. But it always has 'Flatcar' in it.